web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,27 +1,59 @@
 <script lang="ts">
+	import { onMount } from 'svelte';
+	import { lazyLoad } from 'unlazy';
 	import { imageLoad } from '$lib/utils/image-load';
 
 	export let url: string;
 	export let altText: string;
 	export let heightStyle: string | undefined = undefined;
 	export let widthStyle: string;
+	export let thumbhash: string | null = null;
 	export let curve = false;
 	export let shadow = false;
 	export let circle = false;
 	let loading = true;
+
+	let imageElement: HTMLImageElement;
+
+	onMount(() => {
+		if (thumbhash != null) {
+			lazyLoad(imageElement, {
+				hash: thumbhash || '',
```

Comment:
Why the `|| ''` if there's already a check for `null`? What else could the value be here?

Reply:
Someone else commented about this in a review earlier.

The reason for this is unclear to me, but my linter kept yelling at me about null values so I just added that to quell it.

Reply:
If the code is inside of a callback function inside an if (it was in onMount before), then it could be not null when the if statement is evaluated and still be null when the callback was invoked. That's why your linter was complaining, the value could have changed and become null by the time it was evaluated on that line.

Not that you've removed the separation between the if and the use of thumbhash, it's no longer possible for it to be null when it is used and the extra || '' is no longer necessary. You should not have linting errors anymore if you remove it.